crates/matrix-sdk/Cargo.toml

```diff
@@ -58,6 +58,8 @@ experimental-sliding-sync = [
 
 docsrs = ["e2e-encryption", "sqlite", "sso-login", "qrcode", "image-proc"]
 
+widget-api = []
```

Comment:
Can you rename this to `experimental-widget-api` and move it up to the other experimental feature definitions?

Reply:
Makes sense! Done.